day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -127,6 +141,46 @@ ose-app-node01.example.com   Ready                      24m       v1.6.1+5115d70
 ose-app-node02.example.com   Ready                      24m       v1.6.1+5115d708d7
 ----
 
+*Increasing storage with a new disk*
+
+. Evacuate the node xref:evacuating-a-node[following the steps above].
+
+. Stop the `docker` and `atomic-openshift-node` services:
++
+----
+# systemctl stop docker atomic-openshift-node
+----
+
+. Resize the existing disk as desired. This process can be different depending
+on your environment. If you are using a cloud provider, you can detach the disk,
```

Comment:
I forgot to mention that if using LVM (which probably is), it is required to:

* Remove the docker lv (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/lv#LV_remove)
```
lvs
lvremove /dev/docker_vg/docker_lv
```
* Remove the docker vg (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/vg_admin#VG_remove)
```
vgs
vgremove docker_vg
```
* Remove the pv (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration/physvol_admin#PV_remove)
```
pvs
pvremove /dev/<my_previous_disk_device>
```

Otherwise LVM will complain if detaching the disk.